crates/python_api/src/eurocode_1_1_2/section_3.rs

```diff
@@ -0,0 +1,374 @@
+use pyo3::prelude::*;
+use pyo3::wrap_pymodule;
+
+// Import all eurocode_1_1_2 section 3 functions
+use ::openfire::eurocode_1_1_2::section_3::{
+    equation_3_1 as rust_equation_3_1,
+    equation_3_2 as rust_equation_3_2,
+    equation_3_3 as rust_equation_3_3,
+    equation_3_4 as rust_equation_3_4,
+    equation_3_5 as rust_equation_3_5,
+    equation_3_6 as rust_equation_3_6,
+    equation_3_7 as rust_equation_3_7,
+};
+
+// Equation 3.1 module functions
+#[pyfunction]
+/// Net heat flux per unit area of the surface (Equation 3.1).
```

Comment:
```suggestion
/// Net heat flux per unit area of the surface.
```
This is something that is happening a lot, the equation number is repeated directly below the title (which contains the equation number)

Reply:
Addressed.